src/Firehose.Web/Authors/GabrielTalavera.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.ServiceModel.Syndication;
+using System.Web;
+using Firehose.Web.Infrastructure;
+
+namespace Firehose.Web.Authors
+{
+    public class BjornHouben : IAmACommunityMember
```

Comment:
This line needs to be updated with your own name

Reply:
Done, thank you for the review!